docs/contributing/architecture/app-layout.rst

```diff
@@ -65,15 +65,15 @@ This is the basic layout of the ``pulp.app`` package, on the filesystem::
     └── __init__.py
 
 
-The contents of this package are documented in detail in the :doc:`../platform_api/index`
+The contents of this package are documented in detail in the :doc:`../platform-api/index`
 documentation. Details how this package is organized can be found
```

Comment:
"Details how this package is organized" --> Precede "how" with either "on", "about", or maybe "regarding"